### PR TITLE
Updated addresses for the 1.0.2 release, added OSM peek and peep.

### PR DESCRIPTION
**Pull request by EdNoepel**
Added @nanexcool to confirm coolness with my OSM `peek` and `peep`, since `toll` is only for smart contracts consuming the feed, and since everyone else is reading prices in this fashion.

@grandizzy: Please review everything else.  Note our local testnet doesn't have an OSM, so I didn't write a unit test.  Integration tested manually.

### CODE REVIEW

File: pymaker/oracles.py [hunk 1]
```diff
@@ -46,7 +47,11 @@ def __init__(self, web3: Web3, address: Address):
     def poke(self) -> Transact:
         return Transact(self, self.web3, self.abi, self.address, self._contract, 'poke', [])
 
-    def __repr__(self):
-        return f"OSM('{self.address}')"
+    def peek(self) -> Wad:
+        return Wad(Web3.toInt(self.web3.eth.getStorageAt(self.address.address, 3)[16:]))
```
Comment (grandizzy) [inline pymaker/oracles.py]: I'd suggest to we externalize logic of extracting price e.g in `_extract_price(storage)`  function and add unit test for it. Then reuse function in both `peek` and `peep`

Comment (grandizzy) [inline pymaker/oracles.py]: should be returning always a Wad? (that is even in case we add an collateral with different precision?)

Comment (EdNoepel) [reply pymaker/oracles.py]: The contracts don't expose the precision.  Until they do, it's reasonable to treat all collateral types with the same precision, as the `join` contracts do.

Comment (EdNoepel) [reply pymaker/oracles.py]: Done.

Comment (EdNoepel) [reply pymaker/oracles.py]: Done.

Comment (grandizzy) [reply pymaker/oracles.py]: :+1:

File: tests/test_dss.py [hunk 2]
```diff
@@ -542,6 +543,16 @@ def test_drip(self, mcd):
         assert mcd.pot.drip().transact()
 
 
+class TestOsm:
+    def test_price(self, web3, mcd):
```
Comment (grandizzy) [inline tests/test_dss.py]: was thinking more like passing storage value to `_extract_price(storage_value)` but this is cool too :)

**Pull request merged**